pkg/csi/cinder/controllerserver.go

```diff
@@ -653,6 +661,62 @@ func (cs *controllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 	}, nil
 }
 
+func (cs *controllerServer) StartWathcer() error {
+	snapClient, err := snapClient.NewForConfig(cs.RestConfig)
+	if err != nil {
+		klog.ErrorS(err, "Error building snapshot clientset")
+		return err
+	}
+	watcher, err := client.NewWithWatch(cs.RestConfig, client.Options{})
+	if err != nil {
+		klog.ErrorS(err, "Error building watcher")
+		return err
+	}
+	watchInterface, err := watcher.Watch(context.TODO(), &corev1.PersistentVolumeClaimList{})
+	if err != nil {
+		klog.ErrorS(err, "Error watching PersistentVolumeClaim")
+		return err
+	}
+	go func() {
```

Comment:
Can we define this function statically and call it here? e.g. `go watcher(watchInterface)`?